soc/mediatek/mt8xxx/soc.c

```diff
@@ -50,6 +50,27 @@ __asm__(".align 4\n\t"
 	"  movi  a1, " INIT_STACK "\n\t"
 	"  call4 c_boot\n\t");
 
+/* Unfortunately the SOF kernel loader doesn't understand the boot
+ * vector in the ELF/rimage file yet, so we still need a stub to get
```

Comment:
Might be worth to file a https://github.com/thesofproject/sof/issues ticket and point to this.

Reply:
I have a prototype fix already actually, but needs to wait for firmware submission that exploits an arbitrary boot vector.